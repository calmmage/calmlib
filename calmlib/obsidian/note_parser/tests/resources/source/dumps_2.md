- [ ] 001
    - unchecked dump item
- [x] 002
    - checked dump item
- 003
    - regular dump item
- [ ] 004
    - text
- 005
    - [ ] dump text
- [x] 006
    - [ ] text
