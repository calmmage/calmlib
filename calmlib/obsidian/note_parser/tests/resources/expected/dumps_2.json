{
  "items": [
    {
      "type": "dump",
      "text": "001",
      "checked": false,
      "children": [
        {
          "type": "list_item",
          "text": "unchecked dump item"
        }
      ]
    },
    {
      "type": "dump",
      "text": "002",
      "checked": true,
      "children": [
        {
          "type": "list_item",
          "text": "checked dump item"
        }
      ]
    },
    {
      "type": "dump",
      "text": "003",
      "children": [
        {
          "type": "list_item",
          "text": "regular dump item"
        }
      ]
    },
    {
      "type": "dump",
      "text": "004",
      "checked": false,
      "children": [
        {
          "type": "list_item",
          "text": "text"
        }
      ]
    },
    {
      "type": "dump",
      "text": "005",
      "children": [
        {
          "type": "list_item",
          "text": "dump text",
          "checked": false
        }
      ]
    },
    {
      "type": "dump",
      "text": "006",
      "checked": true,
      "children": [
        {
          "type": "list_item",
          "text": "text",
          "checked": false
        }
      ]
    }
  ]
}